models/commons/src/discounted-price/discounted-price-draft/presets/index.ts

```diff
@@ -0,0 +1,3 @@
+export const restPresets = {};
+export const graphqlPresets = {};
+export const compatPresets = {};
```

Comment:
We don't need the `compatPresets` here since this is not a legacy migrated model but a new one.

Reply:
@mario-priceless It seems you removed the presets altogether. What I meant is that we don't need the `compatPresets` object, but we need to keep the other ones.
Even we are not populating them now, the idea is that all data models have a presets prop that can be an empty object today but to have some contents in the future.

Please let me know if I need to provide more context on this 🙏